docs/pages/versions/unversioned/sdk/updates.mdx

```diff
@@ -10,102 +10,128 @@ platforms: ['android', 'ios', 'tvos']
 import APISection from '~/components/plugins/APISection';
 import { APIInstallSection } from '~/components/plugins/InstallSection';
 import { YesIcon, NoIcon } from '~/ui/components/DocIcons';
-import { ConfigReactNative, ConfigPluginExample } from '~/components/plugins/ConfigSection';
+import { Collapsible } from '~/ui/components/Collapsible';
 import { BoxLink } from '~/ui/components/BoxLink';
 import { GithubIcon } from '@expo/styleguide-icons';
 
-The `expo-updates` library allows you to programmatically control and respond to new updates made available to your app.
+## Installation & Configuration
```

Comment:
I had originally removed it since it appears right above it. Still make sense to keep it?
<img width="1140" alt="Screenshot 2024-06-03 at 9 28 40 AM" src="https://github.com/expo/expo/assets/189568/8802509f-3e76-4b2a-9656-5c4113bd6695">

Reply:
I like the thorough description you added in this PR with the changes above:

```md
`expo-updates` is a library that enables your app to manage remote updates to your application code. It communicates with the configured remote update service to get information about available updates.
```

This way, the description helps answer what and why of this library.